docs/source/ko/_toctree.yml

```
@@ -46,8 +46,8 @@
       title: (번역 중) Login and logout
     - local: in_translation
       title: (번역 중) Environment variables
-    - local: in_translation
-      title: (번역 중) Managing local and online repositories
+    - local: package_reference/repository
+      title: 로컬과 온라인 리포지토리 관리하기
```

```suggestion
      title: 로컬 및 온라인 리포지토리 관리
```
본문 제목 태그(line5)와 일치하지 않아, 일치를 위해 더 자연스럽다고 생각한 쪽으로 수정해보았습니다:)